ansible_builder/main.py

```diff
@@ -383,6 +328,10 @@ def prepare_appended_steps(self):
 
         return False
 
+    def prepare_build_context(self):
+        self.steps.extend(BuildContextSteps())
```

Comment:
Maaaybe this step could be conditionally skipped in the case that:
 - there is no user-provided python or system requirements file
 - there is no user-provided galaxy requirements.yml file

But if that's the case, why are you using `ansible-builder` in the first place?